cdap-proto/src/main/java/co/cask/cdap/proto/NamespaceMeta.java

```diff
@@ -23,22 +23,19 @@
  * Represents metadata for namespaces
  */
 public final class NamespaceMeta {
+  // id is always identical to name. We keep this field here because the UI may depend on it.
```

Comment:
I guess I could. The backend is not using it anywhere. Just wondering whether it can cause Gson problems if the UI adds it?